dev-qt/qt6-base/qt6_base-6.7.2.recipe

```diff
@@ -211,8 +211,13 @@ INSTALL()
 		$libDir/cmake/Qt6FbSupportPrivate/Qt6FbSupportPrivateTargets-release.cmake
 	sed -i 's,${_IMPORT_PREFIX}/lib',$developDir/lib, \
 		$libDir/cmake/Qt6DeviceDiscoverySupportPrivate/Qt6DeviceDiscoverySupportPrivateTargets-release.cmake
-	sed -i 's,${_IMPORT_PREFIX}/lib/libQt6ExampleIcons.a',$developDir/lib/libQt6ExampleIcons.a, \
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	sed -i 's,${_IMPORT_PREFIX}/lib/x86/libQt6ExampleIcons.a,${_IMPORT_PREFIX}/develop/lib/x86/libQt6ExampleIcons.a,' \
```

Comment:
You could use $relativeDevelopLibDir instead of the `if`.

Reply:
still leaves the "${_IMPORT_PREFIX}/lib/x86/libQt6ExampleIcons.a" where the change should happen.

Reply:
this should work: `"s,\${_IMPORT_PREFIX}/$relativeLibDir/libQt6ExampleIcons.a,\${_IMPORT_PREFIX}/$relativeDevelopLibDir/libQt6ExampleIcons.a,"` (double quotes so that shell variables are expanded and escaping the `${_IMPORT_PREFIX}` with `\` instead).

Reply:
Bugger, was about to merge ;) Will have to boot 32bit up again ...

Reply:
That did the trick, thanks!